src/genie/libs/parser/iosxe/tests/ShowWlanAll/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,340 @@
+expected_output = {
+    "wlan_names": {
+        "lizzard_Fabric_F_cf6efda4": {
+            "identifier": 17,
+            "ssid": "lizzard",
+            "status": "Enabled",
+            "broadcast_ssid": "Enabled",
+            "advertise_ap_name": "Disabled",
+            "universal_ap_admin": "Disabled",
+            "max_clients_wlan": 0,
+            "max_clients_ap": 0,
+            "max_clients_ap_radio": 200,
+            "okc": "Enabled",
+            "active_clients": 4,
+            "chd_per_wlan": "Enabled",
+            "wmm": "Allowed",
+            "channel_scan_defer_priority": {
+                "priority": [
+                    5,
+                    6
+                ]
+            },
+            "scan_defer_time_msecs": 100,
+            "media_stream_multicast_direct": "Disabled",
+            "ccx_aironet_support": "Disabled",
+            "p2p_blocking_action": "Disabled",
+            "radio_policy": "802.11a only",
+            "local_eap_authentication": "Disabled",
+            "mac_filter_authorization_list_name": "Disabled",
+            "mac_filter_override_authorization_list_name": "Disabled",
+            "dot1x_authentication_list_name": "dnac-list",
+            "dot1x_authorization_list_name": "Disabled",
+            "security": {
+                "dot11_authentication": "Open System",
+                "static_wep_keys": "Disabled",
+                "wifi_protected_access_wpa1_wpa2_wpa3": "Enabled",
+                "wifi_protected_access_wpa": {
+                    "wpa_ssn_ie": "Disabled"
+                },
+                "wifi_protected_access_wpa2": {
+                    "wpa2_rsn_ie": "Enabled",
+                    "wpa2_attributes": {
+                        "mpsk": "Disabled",
+                        "aes": "Enabled",
+                        "ccmp256": "Disabled",
+                        "gcmp128": "Disabled",
+                        "gcmp256": "Disabled",
+                        "randomized_gtk": "Disabled"
+                    }
+                },
+                "wifi_protected_access_wpa3": {
+                    "wpa3_ie": "Disabled"
+                },
+                "auth_key_mgmt": {
+                    "dot1x": "Enabled",
+                    "psk": "Disabled",
+                    "cckm": "Disabled",
+                    "ft_dot1x": "Enabled",
+                    "ft_psk": "Disabled",
+                    "dot1x_sha256": "Disabled",
+                    "psk_sha256": "Disabled",
+                    "sae": "Disabled",
+                    "owe": "Disabled",
+                    "suiteb_1x": "Disabled",
+                    "suiteb192_1x": "Disabled"
+                },
+                "owe_transition_mode": "Disabled",
```

Comment:
below line was missed?
```
    CCKM TSF Tolerance                         : 1000

Reply:
I did miss that! Thanks for catching it! Updated!